lib/rules/require-unicode-regexp.js

```diff
@@ -37,31 +57,65 @@ module.exports = {
 
         messages: {
             addUFlag: "Add the 'u' flag.",
-            requireUFlag: "Use the 'u' flag."
+            addVFlag: "Add the 'v' flag.",
+            requireUFlag: "Use the 'u' flag.",
+            requireVFlag: "Use the 'v' flag."
         },
 
-        schema: []
+        schema: [
+            {
+                type: "object",
+                properties: {
+                    requireFlag: {
+                        type: "string"
```

Comment:
```suggestion
                        enum: ["u", "v"]
```

To make sure that only these values can be provided.